.github/workflows/external-account-integration.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: build
         run: |
           cmake --build "${{runner.temp}}/build" -j 2 \
-            --target rest_internal_internal_external_account_integration_test
+            --target common_internal_external_account_integration_test
```

Comment:
"common" between "internal" and "external"  :-)

Reply:
Better than `internal_internal_external_` ? :shrug:  I think I would like to remove the `common_` prefix in a future PR.